epistrains/solver.py

```diff
@@ -1,3 +1,33 @@
+from typing import List
+import numpy as np
+import scipy.integrate
+from epistrains.strain import Strain
+
 class Solver:
-    def __init__(self):
-        pass
+    """
+    :param strains: a list of strains to be infected
+    :type strains: _type_
+    :param t_eval: a list of arguments for start time, end time and time interval
+    :type t_eeval: list
+    :param time: timeframe (h) over which the models should be solved for, 
+        defaults to 1
+    :type time: float, optional
+    """
+    def __init__(self, strains: List[Strain], time = 1):
+        self.time = round(time, 2)
+        self.solution = []
```

Comment:
I think this should default to `None` to make it clear that it hasn't been solved yet. I'll probably make a new PR soon with some style changes though, so this is fine for now.